_includes/footer.html

```diff
@@ -0,0 +1,5 @@
+		<script src="js/jasmine/src/splitter.js"></script>
+		<script src="js/jasmine/src/displayer.js"></script>
+		<script src="js/jasmine/src/button-action.js"></script>
+		<script src="js/jasmine/src/button.js"></script>
+		<script src="js/main.js"></script>
```

Comment:
I don't think you need to stress over the indentation to perfectly match where it will be injected. I would almost prefer having all lines starting at column 1.